source/_integrations/volvo.markdown

```diff
@@ -190,10 +197,10 @@ Set the **Device class** to **Timestamp** and optionally choose your vehicle for
 
 The **Volvo** integration fetches data from the API at different intervals:
 
-- **Every 60 minutes**: diagnostics, odometer, and statistics
-- **Every 15 minutes**: car connectivity and fuel status
-- **Every 2 minutes**: energy data (for battery cars)
-- **Every minute**: doors and window status
+- **Every 60 minutes**: diagnostics, odometer, and statistics.
+- **Every 15 minutes**: car connectivity, fuel status and location.
```

Comment:
```suggestion
- **Every 15 minutes**: car connectivity, fuel status, and location.
```
we use the [Oxford comma](https://learn.microsoft.com/en-us/style-guide/punctuation/commas) :nerd_face: